PROJECT

```diff
@@ -101,4 +101,13 @@ resources:
   webhooks:
     validation: true
     webhookVersion: v1
+- api:
+    crdVersion: v1
+    namespaced: true
+  domain: eks.amazonaws.com
+  group: anywhere
+  kind: NutanixConfig
```

Comment:
This should be the `Kind` of the types we are adding. So `NutanixDatacenterConfig` and `NutanixMachineConfig`.